nixos/doc/manual/release-notes/rl-2411.section.md

```diff
@@ -68,6 +68,8 @@
 
 - `nginx` package no longer includes `gd` and `geoip` dependencies. For enabling it, override `nginx` package with the optionals `withImageFilter` and `withGeoIP`.
 
+- `nrfutil` which previously pointed to the now-deprecated `pc-nrfutil` python package, has been repackaged under the same name with the new nrfutil tool.
```

Comment:
Maybe it should instead be packaged as `nrf-util` to avoid the nameclash - If it's really different.

That's also what it's called in one url: https://www.nordicsemi.com/Products/Development-tools/nrf-util` and the documentation (with a space)

Then we could add in a `throw` in aliases.nix as well to make it very clear for anyone who's been relying on the old tool (if you think there might be someone out there)

Reply:
It's possible for the binary to have a different name than the package, but I'll defer to your judgement on this.

My point wasn't that the dash was literally part of the name, just that replacing space with a dash is common. `nrfUtil` or `nrf-util` could also work.

The idea being that old consumers can get a `nrfutil (pc-nrfutil) is deprecated please use nrf-util instead` build time error

If the program is user facing and never used in scripts and such it's probably not a big deal